### ISSUE TITLE
a:focus.outline should not be `none` for accesibility/a11y

### ISSUE THREAD

**Issue by cafesanu** [NG-ZORRO/ng-zorro-antd#3494]
### Reproduction link
[https://github.com/NG-ZORRO/ng-zorro-antd/blob/46fbee5bb0f917fd532a2b15b5684e5e316c57ad/components/style/core/base.less#L234](https://github.com/NG-ZORRO/ng-zorro-antd/blob/46fbee5bb0f917fd532a2b15b5684e5e316c57ad/components/style/core/base.less#L234)

### Steps to reproduce
when using the tab, links are not focusable. This would be fine, except that it does not comply with accessibility/a11y. Ideally, we should be able to overwrite this, so, the less variables could have a default additional variable`@a-focus-outline: none;`, and we could add to the line provided in the link:

```less
a {
  ...
  &:focus {
    outline: @a-focus-outline;
  }
}
```
This way, user can overwrite the default variable with our favorite way of focusing.

### What is expected?
`a.outline` to be able to be modifiable

### What is actually happening?
`a.outline` is not modifiable via less variables.

| Environment | Info |
|---|---|
| ng-zorro-antd | 7.4.1 |
| Browser | all |

**Comment by hsuanxyz** [comment]
close via https://github.com/ant-design/ant-design/pull/22511
